src/main/java/de/tum/in/www1/artemis/web/rest/ExerciseResource.java

```diff
@@ -327,5 +328,40 @@ public ExerciseResource(ExerciseRepository exerciseRepository, ExerciseService e
             .header("filename", zipFile.getName())
             .body(resource);
     }
+    /**
+    * GET  /exercises/:id/participations/:studentIds : sends all submissions from studentlist as zip
+        *
+        * @param id the id of the exercise to get the repos from
+        * @param studentIds the studentIds seperated via semicolon to get their submissions
+     * @return ResponseEntity with status
+     */
+    @GetMapping(value = "/exercises/{id}/participations/{studentIds}")
+    @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
+    @Timed
+    public ResponseEntity<Resource> exportSubmissions(@PathVariable Long id, @PathVariable String studentIds) throws IOException {
+        Exercise exercise = exerciseService.findOne(id);
+        Course course = exercise.getCourse();
+        User user = userService.getUserWithGroupsAndAuthorities();
+        if (!authCheckService.isTeachingAssistantInCourse(course, user) &&
+            !authCheckService.isInstructorInCourse(course, user) &&
+            !authCheckService.isAdmin()) {
+            return ResponseEntity.status(HttpStatus.FORBIDDEN).build();
+        }
+        List<String> studentList = Arrays.asList(studentIds.split("\\s*,\\s*"));
+
```

Comment:
What happens if studentIds is null or empty? Could we catch this case earlier and return then to avoid any kind of problem that could arise because of it and to improve performance?